include/z64.h

```diff
@@ -701,6 +701,14 @@ typedef struct {
     }                   restrictions;
 } InterfaceContext; // size = 0x270
 
+typedef enum {
+    /* 0 */ PAUSE_BG_PRERENDER_OFF, // Inactive, do nothing.
+    /* 1 */ PAUSE_BG_PRERENDER_DRAW, // The current frame is only drawn for the purpose of serving as the pause background.
```

Comment:
When reading the code I got confused between this being the `DONE` state, as in I thought `DRAW` here meant "draw the pause background".
I think something like `SETUP` would work here and be less ambiguous.

Reply:
agreed here. in this state `SREG(33)` will decrement the frame buffer index causing the screen to "freeze" and it deliberately  skips the "draw overlay elements" block so that the hud wont show up in the "screenshot". I think its fair to say that this is the setup or "setting up the frame" for the bg before it starts processing

Reply:
I'll change to setup, it is indeed less ambiguous
Is the comment fine?